kornia/optical_flow/lucas_kanade_opticalflow.py

```diff
@@ -0,0 +1,81 @@
+import torch
+import torch.nn as nn
+
+from kornia.filters import filter2d
+
+
+def optical_flow_lk(image_prev: torch.Tensor, image_next: torch.Tensor) -> torch.Tensor:
+    r"""Calculate an optical flow for an input pair of images.
+
+    Args:
+        image_prev: the input tensor with shape :math:`(B,C,H,W)`.
+        image_next: the size of the kernel.
+
+
+
+    Returns:
+        stacked tensor with shape :math:`(B,2,H,W)`.
+
+    """
+    fx1 = torch.Tensor([[-1, 1], [-1, 1]]).unsqueeze(0)
```

Comment:
Could you please use https://kornia.readthedocs.io/en/latest/_modules/kornia/filters/sobel.html#spatial_gradient instead? 'diff' kernel is exactly what you have here

Reply:
Noted. Will work on it. Thanks